tutorials/best_practices/index.rst

```diff
@@ -15,3 +15,5 @@ Best practices
    godot_notifications
    data_preferences
    logic_preferences
+   project_organization
+   version_control_systems
```

Comment:
I'm not sure if this is the best location for those (both with regard to being in "Best practices" and at the end of that section), since both are topics that users should read early on (at least the project organization one, the VCS is most about the Git plugin which is fully optional).

The rest of the Best practices section seems to be fairly advanced stuff that one would want to read after having used Godot a bit, to understand how to go from "hack your way to make it work" to "best practice guidelines" for complex projects.

Maybe these would be worth having in a section about setting up a project? There could also be documentation about some settings that one needs to define when creating a project (renderer, physics backend for 3D, import formats for VRAM compressed textures depending on the target platforms, etc.).

That being said, I'm fine merging this as is and move things again later on if a better option presents itself.

Reply:
Sure, I put them where I felt could fit with the sections we had. Best practices is advanced at the moment mostly because Will wrote it from an engineer's perspective.

To me, the section name itself, "Best practices", is a good fit for content that answer common beginner questions. What are the best practices is one of the topics people request the most, be it for scripting, how to split or organize scenes, files, etc. I think it's also a section name that people will click. But there's work to do on the section for that to happen.

A project setup section could work. Note that we already have things covered in relevant sections: rendering for the renderers, physics backend in physics... they fit well in there leaving a limited set of topics for a project setup section.